File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.java
class Solution {
    public int solution(int n) {
        int answer = 0; //피자 개수
        
        if(n%7 == 0) {
            answer = n / 7;
        } else {
            answer = n / 7 + 1;
        }
        
        
        
        
        
        
        
        return answer;
    }
}

File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.js
function solution(n) {
    var answer = 0;
    if(n >= 7) {
        if(n % 7 == 0) {
            answer = parseInt(n / 7);
        } else {
            answer = parseInt(n / 7) + 1;
        }
        
    } else {
        answer = 1;
    }
    
    return answer;
}